specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -121,7 +120,7 @@ namespace Azure.Messaging.EventGrid {
   }
    
   @doc("Details of the LockTokens information. This is used for both Acknowledge and Release operation response.")
-  model LockTokensResponse {
+  model LockTokensResult {
```

Comment:
This should really be something like `SettlementResult`. It contains the result from the settlement operation - either Acknowledge or Release.